.rubocop_todo.yml

```diff
@@ -1,44 +1,110 @@
 # versions of RuboCop, may require this file to be generated again.
 
-Metrics/AbcSize:
-  Max: 16
+# Offense count: 2
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyleAlignWith, AutoCorrect, Severity.
+# SupportedStylesAlignWith: keyword, variable, start_of_line
+Layout/EndAlignment:
   Exclude:
     - 'lib/ossert/fetch/github.rb'
 
+# Offense count: 2
+# Configuration parameters: AllowSafeAssignment.
+Lint/AssignmentInCondition:
+  Exclude:
+    - 'lib/ossert/fetch/stackoverflow.rb'
+    - 'lib/ossert/workers/partial_fetch.rb'
+
+# Offense count: 2
+Lint/DuplicateMethods:
+  Exclude:
+    - 'lib/ossert/classifiers/decision_tree.rb'
+    - 'lib/ossert/quarters_store.rb'
+
+# Offense count: 1
+# Configuration parameters: IgnoreImplicitReferences.
+Lint/ShadowedArgument:
+  Exclude:
+    - 'lib/ossert/repositories.rb'
+
+# Offense count: 27
+Metrics/AbcSize:
+  Max: 150
+
+# Offense count: 17
+# Configuration parameters: CountComments, ExcludedMethods.
 Metrics/BlockLength:
+  Max: 412
+
+# Offense count: 5
+# Configuration parameters: CountComments.
+Metrics/ClassLength:
+  Max: 344
+
+# Offense count: 2
+Metrics/CyclomaticComplexity:
+  Max: 16
+
+# Offense count: 19
+# Configuration parameters: CountComments.
+Metrics/MethodLength:
+  Max: 55
+
+# Offense count: 2
+Metrics/PerceivedComplexity:
+  Max: 17
+
+# Offense count: 1
+Naming/MemoizedInstanceVariableName:
+  Exclude:
+    - 'lib/ossert/fetch/rubygems.rb'
+
+# Offense count: 2
+# Configuration parameters: MinNameLength, AllowNamesEndingInNumbers, AllowedNames, ForbiddenNames.
+# AllowedNames: io, id
+Naming/UncommunicativeMethodParamName:
   Exclude:
-    - 'ossert.gemspec'
-    - 'lib/ossert/tasks/ossert.rake'
-    - 'lib/ossert/tasks/database.rake'
     - 'spec/ossert_spec.rb'
-    - 'spec/spec_helper.rb'
 
-Metrics/ClassLength:
+# Offense count: 3
+Style/DateTime:
   Exclude:
     - 'lib/ossert/fetch/github.rb'
+    - 'lib/ossert/quarters_store.rb'
+    - 'lib/ossert/repositories.rb'
 
-Metrics/CyclomaticComplexity:
+# Offense count: 50
+Style/Documentation:
+  Enabled: false
+
+# Offense count: 2
+# Configuration parameters: MinBodyLength.
+Style/GuardClause:
   Exclude:
-    - 'lib/ossert/fetch/github.rb'
+    - 'lib/ossert/fetch/stackoverflow.rb'
 
-Metrics/MethodLength:
+# Offense count: 9
+# Cop supports --auto-correct.
+Style/IfUnlessModifier:
   Exclude:
+    - 'lib/ossert/classifiers/growing/classifier.rb'
     - 'lib/ossert/fetch/github.rb'
+    - 'lib/ossert/fetch/stackoverflow.rb'
 
-Metrics/PerceivedComplexity:
-  Max: 16
+# Offense count: 1
+# Cop supports --auto-correct.
+Style/MutableConstant:
   Exclude:
-    - 'lib/ossert/fetch/github.rb'
+    - 'config/sidekiq.rb'
 
+# Offense count: 261
+# Configuration parameters: AllowHeredoc, AllowURI, URISchemes, IgnoreCopDirectives, IgnoredPatterns.
+# URISchemes: http, https
 Metrics/LineLength:
-  Max: 120
-
-Style/Documentation:
-  Enabled: false
+  Max: 240
```

Comment:
Hmm. I don't like such large lines. Maybe we could Exclude some files and enforce previous value.